modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/provider/RedirectProviderImplTest.java

```diff
@@ -193,11 +196,112 @@ public void testRewritePatternSingleMismatch() {
 				new RedirectPatternEntry(
 					Pattern.compile("^a(b)c"), "x$1z", StringPool.BLANK)));
 
-		Assert.assertNull(_getRedirectProviderRedirect("123"));
+		Assert.assertNull(
+			_getRedirectProviderRedirect("123", StringPool.BLANK));
+
+		_verifyMockInvocations();
+	}
+
+	@Test
+	public void testSimplePatternMatchNoUserAgent() {
```

Comment:
Shouldn't this be `testSimplePatternMatchesNoUserAgent`? "Match" is a noun, not a verb. Same with the rest.

Reply:
Note that depending on the context "match" may be correct (e.g. when the method name refers to a "match" as a thing, not as an action).